api/src/main/java/org/openmrs/scheduler/db/SchedulerDAO.java

```diff
@@ -39,6 +39,15 @@
 	 */
 	public TaskDefinition getTask(Integer taskId) throws DAOException;
 	
+	/**
+	 * Get task by UUID
+	 * 
+	 * @param uuid unique task identifier
+	 * @return task with given universal unique identifier
```

Comment:
Instead of duplicating the same javadoc, we use `see also`

Reply:
i have fixed it

Reply:
hello @dkayiwa if all is well on this issue then we can merge, so that i can start working on the dependent issue to 
Thanks